src/components/ToyContainer.js

```diff
@@ -1,10 +1,11 @@
-import React from "react";
-import ToyCard from "./ToyCard";
+import React from "react"
+import ToyCard from "./ToyCard"
 
-function ToyContainer() {
-  return (
-    <div id="toy-collection">{/* Render the collection of ToyCards */}</div>
-  );
+function ToyContainer({ toys, onDeleteToy }) {
+  const displayedToys = toys.map((toy) => {
+    return <ToyCard key={toy.id} toy={toy} onDeleteToy={onDeleteToy} />
+  })
+  return <div id="toy-collection">{displayedToys}</div>
```

Comment:
A space between these lines would be great. 
```suggestion
  const displayedToys = toys.map((toy) => {
    return <ToyCard key={toy.id} toy={toy} onDeleteToy={onDeleteToy} />
  })
  
  return <div id="toy-collection">{displayedToys}</div>
```